Source/MockException.cs

```diff
@@ -176,23 +178,23 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
 #endif
 	internal class MockVerificationException : MockException
 	{
-		IProxyCall[] failedSetups;
+		IEnumerable<IProxyCall> failedSetups;
 
-		public MockVerificationException(IProxyCall[] failedSetups)
+		public MockVerificationException(IEnumerable<IProxyCall> failedSetups)
```

Comment:
This change is to avoid an unnecessary call to `.ToArray()` in the calling code.